src/examples/blinkm/blinkm.go

```diff
@@ -0,0 +1,38 @@
+// Connects to an BlinkM I2C RGB LED.
+package main
+
+import (
+	"machine"
+	"time"
+)
+
+func main() {
+	machine.I2C0.Configure(machine.I2CConfig{})
+
+	// Init BlinkM
+	machine.I2C0.WriteTo(0x09, []byte("o"))
+
+	version := []byte{0, 0}
+	machine.I2C0.WriteTo(0x09, []byte("Z"))
+	machine.I2C0.ReadFrom(0x09, version)
+	println("Firmware version:", string(version[0]), string(version[1]))
+
+	count := 0
+	for {
+		machine.I2C0.WriteTo(0x09, []byte("n"))
+
+		switch count {
+		case 0:
+			machine.I2C0.WriteTo(0x09, []byte{0x99, 0xff, 0x00})
```

Comment:
Suggestion: add a comment what color you're sending with these commands (I'm guessing yellow in this case?).

Reply:
Actually I had just made up some numbers. I will chose a real color.